extensions/vertx-http/deployment/src/main/resources/dev-templates/index.html

```diff
@@ -1,46 +1,15 @@
 {#include main}
-  {#style}
-.card.extension {
-    background-color: white;
-    box-shadow: 0 0 0 1px #d4d4d5, 0 2px 4px 0 rgba(34,36,38,.12), 0 2px 10px 0 rgba(34,36,38,.15);
-    border-radius: 0.25em;
-    position: relative;
-}
-.card.extension .card-header {
-    background-color: #f8f8f8;
-    padding: 0.75rem 0.75rem;
-    margin-bottom: 0;
-    border-bottom: 1px solid rgba(0,0,0,.125);
-}
-.card.extension .card-header h5 {
-    margin-bottom: 0px;
-}
-.card.extension .card-text {
-    color: rgba(0,0,0,.6);
-}
-hr {
-    margin-top: 0;
-}
-.btn.btn-guide {
-    font-size: 0.7em;
-    padding-top: 0.1em;
-    padding-bottom: 0.1em;
-    padding-left: 0.4em;
-    padding-right: 0.4em;
-}
-{/style}
```

Comment:
Are all these styles in the stylesheet now?

Reply:
No, it's either not used anymore or been replaced with built in bootstrap classes.

Reply:
@gsmet let me know when I can mark as ready for review